etc/config.xml

```diff
@@ -17,6 +17,8 @@
                 <webhook_triggered_at>0</webhook_triggered_at>
                 <enable_pending_orders_cron>1</enable_pending_orders_cron>
                 <pending_orders_timeout>30</pending_orders_timeout>
+                <enable_pending_orders_age>0</enable_pending_orders_age>
```

Comment:
@rohitcbr , why this one required?

Reply:
@ramth05 If pending order age is enabled, we are using the pending order age value as upper limit and timeout value as lower limit in search criteria filter. If it is disabled, we are using the existing search criteria containing only the timeout value as filter. By default, keeping pending order age disabled.